src/wp-includes/theme-previews.php

```diff
@@ -75,10 +75,12 @@ function wp_block_theme_activate_nonce() {
 	<?php
 }
 
-// Attaches filters to enable theme previews in the Site Editor.
-if ( ! empty( $_GET['wp_theme_preview'] ) ) {
-	add_filter( 'stylesheet', 'wp_get_theme_preview_path' );
-	add_filter( 'template', 'wp_get_theme_preview_path' );
-	add_action( 'init', 'wp_attach_theme_preview_middleware' );
-	add_action( 'admin_head', 'wp_block_theme_activate_nonce' );
+// Attaches filters and actions to enable Block Theme Previews in the Site Editor.
```

Comment:
This would need a (proper) docblock. Maybe something like:
```suggestion
/**
 * Attaches filters and actions to enable Block Theme Previews in the Site Editor.
 *
 * The filters and actions have to be added after `pluggable.php` is included as they may
 * trigger code that uses `current_user_can()` which requires functionality from `pluggable.php`.
 *
 * @since 6.3.2
 */
```